circle.yml

```diff
@@ -1,36 +1,210 @@
+        steps:
+            - checkout
+            - run:
+                name: Place the proper Dockerfile
+                command: cp docker/develop-ci/Dockerfile .
+            - setup_remote_docker
+            - run:
+                name: log into Docker
+                command: |
+                    docker login -e $DOCKER_EMAIL -u $DOCKER_USER -p $DOCKER_PASS
+            - run:
+                name: Build Docker container
+                command: docker build --rm=false -t cyclus/cycamore:latest .
+            - run:
+                name: Push on DockerHub
+                command: docker push cyclus/cycamore:latest # push to docker depot
+
+    deploy_stable: # Cyclus/master -> Cyclus:stable
+        docker:
+            - image: circleci/ruby:2.4-node
+        working_directory: ~/cycamore
+        steps:
+            - checkout
+            - run:
+                name: Place the proper Dockerfile
+                command: cp docker/master-ci/Dockerfile .
+            - setup_remote_docker
+            - run:
+                name: Log on DockerHub
+                command: |
+                  docker login -e $DOCKER_EMAIL -u $DOCKER_USER -p $DOCKER_PASS
+            - run:
+                name: Build Docker container
+                command: docker build -t cyclus/cycamore:stable .
+            - run:
+                name: Push on DockerHub
+                command: docker push cyclus/cycamore:stable # push to docker depot
+
+
+    # Debian package generation (on master update)
+    deb_generation:
+        docker:
+            - image: circleci/ruby:2.4-node
+        working_directory: ~/cycamore
+        steps:
+            - checkout
+            - setup_remote_docker
+            - run:
+                name: Tag and Push on DockerHub
+                no_output_timeout: "20m"
+                command: |
+                  docker/deb-ci/build_upload_deb.sh 14
+                  docker/deb-ci/build_upload_deb.sh 16
+    
+
+    # Triggered by Cyclus               
+    dev_against_cyclus_master:  ## Cycamore/dev against Cyclus/master
+        docker:
+            - image: cyclus/cyclus:master
+        working_directory: ~/cycamore
+        steps:
+            - checkout
+            - run:
+                name: Build Docker Image
+                command: |
+                    python install.py -j 2 --build-type=Release \
+                    -DBLAS_LIBRARIES="/opt/conda/lib/libblas.so" \
+                    -DLAPACK_LIBRARIES="/opt/conda/lib/liblapack.so"
+            - run:
+                name: Unit Test
+                command: /root/.local/bin/cycamore_unit_tests; exit $?
+            - run:
+                name: Install nosetest
+                command: pip install nose
+            - run:
+                name: Nosetests
+                command: nosetests -w ~/cycamore/tests; exit $?
+
+
+    # External Triggers
+    cymetric_dev: ## Cymetric/dev against Cyclus/dev + Cyacamore/dev
```

Comment:
Why are we running cymetric stuff here? Cycamore shouldn't depend on cymetric.

Reply:
@scopatz 
the goal is to have a test that provide the status of the Cymetric, when we update Cycamore, just as a warning, to know that change in cycamore broke cymetric. So we can fix cymetric.
the idea is to avoid the situation we run into last year, when I did the release and we released that Cymetric/develop was not compatible with Cyclus/develop, so we(you) had to fix this after the release

Reply:
Ahh I see.  I haven't gotten around to updating the CI and dockerfile for cymetric yet to work with the latest versions for cyclus and cycamore.  I suspect that they are incompatible. I'll try to do that in an hour or so!

Reply:
thx !